tests/acceptance/features/apiGraphUserGroup/editUser.feature

```diff
@@ -37,7 +37,7 @@ Feature: edit user
     Examples:
       | action description           | user    | http-status-code | new-user |
       | change to a valid user name  | Lionel  | 200              | Lionel   |
-      | user name characters         | a*!_+-& | 200              | a*!_+-&  |
+      | user name characters         | a*!_-&  | 200              | a*!_-&   |
```

Comment:
test scenario for `+` will be done in https://github.com/owncloud/ocis/issues/11032